ocs_ci/ocs/platform_nodes.py

```diff
@@ -312,6 +312,21 @@ def restart_nodes(self, nodes, wait=True, force=True):
         )
         self.aws.restart_ec2_instances(instances=instances, wait=wait, force=force)
 
+    def terminate_nodes(self, nodes, wait=True):
+        """
+        Terminate EC2 instances
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to terminate,
+            False otherwise
+        """
```

Comment:
```suggestion
        Terminate EC2 instances
        
        Args:
            nodes (list): The OCS objects of the nodes
            wait (bool): True for waiting the instances to terminate,
                False otherwise
            
        """
```